config/locales/mailers/en.yml

```diff
@@ -248,3 +249,21 @@ en:
         html: "For more information, visit our %{faq_link}."
         text: "For more information, visit our Collections and Challenges FAQ: %{faq_url}"
       faq_link_text: "Collections and Challenges FAQ"
+
+    invited_to_collection_notification:
+      subject: "[%{app_name}][%{collection}] Request to include work in a collection"
+      greeting: "Dear %{name},"
+      part1: "The collection maintainers of %{collection} would like to include your work (%{work}) in their collection!"
+      bye: "Thanks!"
+      text:
+        part2: "To approve or reject this request, please visit your \"Collection Items\" page: %{collection_items_url}."
+        part3: "If in future you would prefer to automatically approve requests to add your works to collections, please edit your Preferences (%{link}) and tick the box next to \"Automatically agree to your work being collected by others in the Archive.\"."
+      html:
+        part2: "To approve or reject this request, please visit your %{link}."
+        link_text: "Collection Items page"
+        part3: "If in future you would prefer to automatically approve requests to add your works to collections, please edit your %{prefs} and tick the box next to: %{quote}"
+        preferences: "Preferences"
+        quote_link: "Automatically agree to your work being collected by others in the Archive."
+
+    added_to_collection_notification:
```

Comment:
Oh, hm, I think this might've slipped in by mistake? When I read the PR title, I assumed it was one mailer that got used in multiple ways, but when I saw this, I double checked and noticed added_to_collection_notification and invited_to_collection_notification are in fact two distinct mailers.

Reply:
It's two mailers in one PR, because they're similar and have a lot of overlap.

Reply:
That's fine, but in that case, shouldn't it also include the text of the email? I'm only seeing the subject for some reason, and no changes to either of the added_to_collection_notification views.

Reply:
You're right, I seem to have missed out the whole rest of it. Shall I split out into two separate pulls/JIRAs, or update this one?

Reply:
Updating this one is fine, I think. It should be fairly painless to test both emails.